### DISCUSSION TITLE
Page view API with Google import

### DISCUSSION THREAD

**Discussion by jsm350** [plausible/analytics#2655] [Anything Else]
When using the pageview API it doesn't include views imported from Google. Is there a way to include those imported views? Thanks.

**Comment by metmarkosaric** [comment]
hi @jsm350! the imported GA data is not part of our stats API nor our CSV export unfortunately so the only way to get those stats is in the dashboard itself

**Reply by jsm350** [reply]
Okay thank you.

**Reply by rodrigotassinari** [reply]
hi @metmarkosaric is there any plan (if possible) to change this? it would be extremely helpful to migrate all our Googla Analytics data to Plausible and be able to have a unified API to query both old (GA) data and new (Plausible) date in just one service.

or even a new, different API endpoint for the imported data would still be helpful.

**Reply by metmarkosaric** [reply]
hi @rodrigotassinari! we'd like to make the imported data become part of our stats api in the future but i cannot make any promise on when. we're a small team with limited resources and are currently working on some more frequently requested features such as funnels. we'll post here when there's some progress on this
